site/config.yaml

```diff
@@ -0,0 +1,58 @@
+baseURL: "https://kubeapps-dev.netlify.app/"
+languageCode: "en-us"
+title: "Kubeapps"
+theme: "template"
+outputs:
+  home: [ "HTML", "REDIRECTS" ]
+pygmentsCodefences: true
+pygmentsStyle: "pygments"
+markup:
+  highlight:
+    anchorLineNos: false
+    codeFences: true
+    guessSyntax: true
+    hl_Lines: ""
+    lineAnchors: ""
+    lineNoStart: 1
+    lineNos: false
+    lineNumbersInTable: true
+    noClasses: true
+    style: friendly
+    tabWidth: 4
+## Only used if use_advanced_docs=false
+# menu:
+#   docs:
+#   - name: Overview
+#     url: /docs/
+params:
+  author: Kubeapps
+  description: Kubeapps website
+  docs_latest: latest
+  docs_right_sidebar: true
+  docs_search: false
+  docs_search_api_key: api_key
+  docs_search_index_name: index_name
+  docs_versioning: true
```

Comment:
The search feature is pending, as we will use an external indexing system that requires us to be "production ready" first.